src/app/_classes/authinjector/authinjector.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {
```

Comment:
Please investigate why there are generated source files here, I would be expecting TypeScript.

Reply:
I was curious about this too, perhaps we should remove these files from the branch for now and push what we need. Figure out what happened later?